client/packages/system/src/Patient/hooks/usePatientModalStore.ts

```diff
@@ -0,0 +1,31 @@
+import { zustand } from '@openmsupply-client/common';
+import { PatientModal } from '../DetailView';
+
+interface PatientModalState {
+  current: PatientModal | undefined;
+  documentName: string | undefined;
+  documentType: string | undefined;
+  reset: () => void;
+  setCurrent: (current: PatientModal | undefined) => void;
+  setDocumentName: (documentName: string | undefined) => void;
+  setDocumentType: (documentType: string | undefined) => void;
```

Comment:
Maybe a comment about the state and what it does would be good. For example, that it manages the current document to be displayed in a modal...